imports/plugins/core/layout/client/templates/layout/footer/footer.html

```diff
@@ -1,7 +1,88 @@
 <template name="layoutFooter">
-  <nav class="navbar-bottom" role="navigation">
-    <div class="row">
-
+    <div class="footer" style="background-color: #255c75; color: white; padding: 3%">
```

Comment:
Please ensure you use 2 spaces for any html file, instead of 4

Reply:
fixed.